ext/node_resolver/resolution.rs

```diff
@@ -597,7 +597,7 @@ impl<TEnv: NodeResolverEnv> NodeResolver<TEnv> {
           for key in imports.keys() {
             let pattern_index = key.find('*');
             if let Some(pattern_index) = pattern_index {
-              let key_sub = &key[0..=pattern_index];
+              let key_sub = &key[0..pattern_index];
```

Comment:
Slight bug copying the code from https://github.com/nodejs/node/blob/f9f9a421a8d3826ade3087a58c4c8535b8af62d0/lib/internal/modules/esm/resolve.js#L707